index.js

```diff
@@ -52,33 +57,31 @@ function parseTweet(tweetObj) {
 
 function processHashTags(tags, tweetObj) {
   tags.forEach((tagObj) => {
-    var anchor = ('#' + tagObj.text).link('http://twitter.com/hashtag/' + tagObj.text);
-    tweetObj.html = tweetObj.html.replace('#' + tagObj.text, anchor);
+    var anchor = ('#' + tagObj.tag).link('http://twitter.com/hashtag/' + tagObj.tag);
+    tweetObj.html = tweetObj.html.replace('#' + tagObj.tag, anchor);
   });
 }
 
 function processSymbols(symbols, tweetObj) {}
 
 function processUserMentions(users, tweetObj) {
   users.forEach((userObj) => {
-    var anchor = ('@' + userObj.screen_name).link('http://twitter.com/' + userObj.screen_name);
-    var regex = new RegExp('@' + userObj.screen_name, 'gi' );
+    var anchor = ('@' + userObj.username).link('http://twitter.com/' + userObj.username);
+    var regex = new RegExp('@' + userObj.username, 'gi' );
     tweetObj.html = tweetObj.html.replace(regex, anchor);
   });
 }
 
 function processUrls(urls, tweetObj) {
-  urls.forEach((urlObj, index) => {
-    var quotedTweetHtml = '';
-    var indices = urlObj.indices;
-    var urlToReplace = tweetObj.text.substring(indices[0],indices[1]);
-
-    if(index === urls.length-1 && tweetObj.quoted_status) {
-      quotedTweetHtml = parseTweets(tweetObj.quoted_status).html;
-      quotedTweetHtml = `<div class="quoted-tweet">${quotedTweetHtml}</div>`
-    }
+  urls.forEach((urlObj) => {
+    var start = urlObj.start;
+    var end = urlObj.end;
+    var urlToReplace = tweetObj.text.substring(start,end);
+
+    // TODO: if `urlObj.expanded_url` is another tweet, we could expand
+    // it into some html that shows the embedded tweet.
```

Comment:
WIP here: https://github.com/blessanm86/tweet-to-html/pull/17, but I'm not sure it's worth it to do this (at least for my use case)